### PR TITLE
EZP-31632: Floating table toolbar

### PR DESCRIPTION
**Pull request by mateuszdebinski**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-31632](https://issues.ibexa.co/browse/EZP-31632)
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   |no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

This PR is port from this PR: https://github.com/ezsystems/ezplatform-richtext/pull/148

#### Checklist:
- [ ] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base-table.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import AlloyEditor from 'alloyeditor';
 
-export default class EzConfigTableBase {
+export default class EzConfgiFixedBase {
```
Comment (dew326) [inline src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base-table.js:3]: Something is wrong here.

**Review by dew326 — CHANGES_REQUESTED** [review]

File: src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base-table.js (hunk 1 above)
Comment (mateuszdebinski) [reply src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base-table.js:3]: no, in 2.5 this class has a bug in the name, please see in src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base-fixed.js

Comment (dew326) [reply src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base-table.js:3]: I don't mean the typo.
You are changing the class name not extend the class with `EzConfgiFixedBase`, or it's too little coffee for me?

Comment (mateuszdebinski) [reply src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base-table.js]: Yes, you are right!, I tested this, and worked even when I changed the name class. I need more coffee :), corrected

File: src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base-table.js [hunk 2]
```diff
@@ -24,4 +27,8 @@ export default class EzConfigTableBase {
     getConfigName() {
         return '';
     }
+
+    getArrowBoxClasses() {
```
Comment (dew326) [inline src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base-table.js:31]: the `setPosition` and `getArrowBoxClasses` should be removed from the constructor as it is in the original PR

Comment (mateuszdebinski) [reply src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base-table.js:28]: I removed it early, but when rollback this file I forgot to remove it again

File: src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base-table.js [hunk 3]
```diff
@@ -1,7 +1,10 @@
 import AlloyEditor from 'alloyeditor';
```
Comment (dew326) [inline src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base-table.js:1]: Now, this import seems unused.

Comment (mateuszdebinski) [reply src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base-table.js]: Yes, I checked again now and I think all was now moved from the original PR

**Review by dew326 — APPROVED** [review]

**Review by lucasOsti — APPROVED** [review]

**Review by GrabowskiM — APPROVED** [review]

**Review by micszo — APPROVED** [review]
QA Approved on eZ Platform EE v2.5.17 with diff.
Also checked https://issues.ibexa.co/browse/EZP-32006 and https://issues.ibexa.co/browse/EZP-32187 (does not affect 2.5).
Performed sanities for block and inline custom tags.

**Comment by lserwatka** [comment]
Could you merge it up?

**Comment by mateuszdebinski** [comment]
Done, merge up to 2.2, 2.3 and master

**Pull request merged**